by-language/java-jooq/src/test/java/io/crate/demo/jooq/ApplicationTest.java

```diff
@@ -40,7 +40,7 @@ public void testExampleWithDynamicSchema() throws SQLException, IOException {
 
         // Check number of records.
         DSLContext db = app.getDSLContext();
-        int count = db.fetchCount(DSL.selectFrom("testdrive.book"));
```

Comment:
Maybe a few more advanced queries would be nice.

Reply:
Thanks. I've planned to exercise more advanced queries within a subsequent iteration.

Reply:
👍

Reply:
The [backlog] now enumerates a number of items which may be addressed on behalf of later iterations.

[backlog]: https://github.com/crate/cratedb-examples/blob/399da690b/by-language/java-jooq/backlog.rst